ui-events-winit/CHANGELOG.md

```diff
@@ -13,15 +13,34 @@ You can find its changes [documented below](#010-2025-05-08).
 
 ## [Unreleased]
 
-This release has an [MSRV][] of 1.73.
+This release has an [MSRV][] of 1.81.
+
+### Added
+
+* `PointerGesture` and `PointerGestureEvent` types, with `Gesture` variant added to `PointerEvent`. ([#80][] by [@xorgy][] and [@arthur-fontaine][])
+* `scale_factor` parameter to `WindowEventReducer::reduce` for device-independent slop in tap detection. ([#78][] by [@xorgy][])
+
+### Changed
+
+* Convert `PointerEvent` struct variants (`Down`, `Up`, `Scroll`) to separate structs. ([#63][] by [@nicoburns][])
+* Reduce allocations in `TapCounter`. ([#61][] by [@nicoburns][])
+
+[@arthur-fontaine]: https://github.com/arthur-fontaine
+[@nicoburns]: https://github.com/nicoburns
+[@xorgy]: https://github.com/xorgy
+
+[#61]: https://github.com/endoli/ui-events/pull/61
+[#63]: https://github.com/endoli/ui-events/pull/63
+[#78]: https://github.com/endoli/ui-events/pull/78
+[#80]: https://github.com/endoli/ui-events/pull/80
 
 ## [0.1.0][] - 2025-05-08
 
-This release has an [MSRV][] of 1.73.
+This release has an [MSRV][] of 1.81.
 
 This is the initial release.
 
 [Unreleased]: https://github.com/endoli/ui-events/compare/v0.1.0...HEAD
-[0.1.0]: https://github.com/linebender/color/releases/tag/v0.1.0
+[0.1.0]: https://github.com/endoli/ui-events/releases/tag/v0.1.0
```

Comment:
I should have caught this on the initial review. :see_no_evil:  @waywardmonkeys

Reply:
SHAME SHAME